l2/system-contracts/DefaultAccount.sol

```diff
@@ -17,7 +17,7 @@ import './interfaces/IAccount.sol';
 contract DefaultAccount is IAccount {
 	using TransactionHelper for *;
 
-	// bytes4(keccak256("isValidSignature(bytes32,bytes)")
+	/// @dev bytes4(keccak256("isValidSignature(bytes32,bytes)"))
 	bytes4 constant EIP1271_SUCCESS_RETURN_VALUE = 0x1626ba7e;
```

Comment:
why not use the actual code instead of having it commented?

Reply:
We thought that constants are not considered in compile time, though that was true for the older issue, solidity ^0.8.0 work well. Will change in the next release!